tools/morpheus/morpheus.xml

```diff
@@ -1,49 +1,11 @@
-<tool id="morpheus" name="Morpheus" version="2.255.0">
+<tool id="morpheusid" name="Morpheus" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
```

Comment:
please don't change the tool ID, otherwise we loose the tool lineage

Reply:
My bad. I forgot to change it back after testing.